contribs/application/src/main/java/org/matsim/application/prepare/freight/tripExtraction/ExtractRelevantFreightTrips.java

```diff
@@ -144,8 +144,8 @@ public Integer call() throws Exception {
             Coord endCoord = endActivity.getCoord();
             double departureTime = startActivity.getEndTime().orElse(0);
 
-            boolean originIsInside = relevantArea.contains(MGC.coord2Point(startCoord));
-            boolean destinationIsInside = relevantArea.contains(MGC.coord2Point(endCoord));
+			boolean originIsInside = relevantArea.contains(MGC.coord2Point(sct.transform(startCoord)));
```

Comment:
The indentation is now messed up probably mixed spaces and tabs,

Reply:
I changed the indentations